fastlane/Fastfile

```diff
@@ -136,14 +136,48 @@ lane :merge_main_to_develop do
   sh('git push')
 end
 
+desc 'Compresses the XCFrameworks into zip files'
+lane :compress_frameworks do
+  Dir.chdir('..') do
+    FileUtils.cp('LICENSE', 'Products/LICENSE')
+    Dir.chdir('Products') do
+      ['StreamChat', 'StreamChatUI'].each do |framework|
+        sh("zip -r #{framework} ./#{framework}.xcframework ./LICENSE")
+        sh("swift package compute-checksum #{framework}.zip")
+      end
+      sh('zip -r "StreamChat-All" ./*.xcframework ./LICENSE')
+    end
+  end
+end
+
+desc 'Cleans Products and DerivedData folders'
+lane :clean_products_and_derived_data do
+  Dir.chdir('..') do
+    FileUtils.rm_rf(
+      %w[
+        DerivedData
+        Products/*.xcframework
+        Products/*.bundle
+        Products/*.BCSymbolMaps
+        Products/*.dSYMs
+        Products/LICENSE
+      ]
+    )
+    FileUtils.mkdir_p(%w[Products DerivedData])
+  end
+end
+
 desc 'Completes an SDK Release'
 lane :publish_release do |options|
   version = options[:version] || ''
   UI.user_error!('You need to pass the version of the release') unless version.length > 0
 
   # Create XCFrameworks
-  xcversion(version: xcode_version)
-  sh('cd .. && make frameworks')
+  xcversion(version: '13.0')
+  clean_products_and_derived_data
+  build_xcframeworks
```

Comment:
Missing sending the logs to a file here, otherwise it might get too verbose

Reply:
I returned the log even tho it never worked as we cleared DerivedData as a post condition of `make frameworks` command. So now it will be saved to `fastlane/fastlane.log`